doc/classes/DirAccess.xml

```diff
@@ -248,6 +248,14 @@
 				[b]Note:[/b] This method is implemented on macOS, Linux (for EXT4 and F2FS filesystems only) and Windows. On other platforms, it always returns [code]true[/code].
 			</description>
 		</method>
+		<method name="is_equivalent" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="path_a" type="String" />
+			<param index="1" name="path_b" type="String" />
+			<description>
+				Returns [code]true[/code] if paths [param path_a] and [param path_b] resolve to the same file system object. Returns [code]false[/code] otherwise, even if the files are bit-for-bit identical (e.g., identical copies of the file that are not symbolic links).
```

Comment:
I'd suggest to add a `[codeblock]` with a few examples that make it clear what this is useful for.
```suggestion
				Returns [code]true[/code] if paths [param path_a] and [param path_b] resolve to the same file system object (taking into account case sensitivity, relative paths, and symbolic links). Returns [code]false[/code] otherwise, even if the files are bit-for-bit identical (i.e. identical copies of a file that are not symbolic links).
```

Reply:
Added this change and a few examples in a separate PR, since this is already merged - https://github.com/godotengine/godot/pull/104767